README.md

```diff
@@ -117,13 +122,13 @@ We use clang-tidy 14 as the linter.
 
 ### Install
 
-#### Ubuntu 22.04
+#### Debian Bookworm
```

Comment:
I think we could also change Coverage section in this readme to Debian instead of Ubuntu.